K2Bridge/Visitors/Aggregations/AggregationDictionaryVisitor.cs

```diff
@@ -70,6 +71,47 @@ public string BuildBucketAggregationQuery(BucketAggregation bucketAggregation, B
             query.Append($"{KustoQLOperators.CommandSeparator} {KustoQLOperators.Summarize} {bucketAggregation.SubAggregationsKustoQL}");
             query.Append($"{definition.BucketExpression};");
 
+            // Create a table named "metadata", with one column per aggregation and dynamic value.
```

Comment:
I am missing a bit more elaboration here. other wise looks good

Reply:
Can you check the comment for BuildMetadataQuery()? It should clarify.